lib/client-assets.php

```diff
@@ -566,7 +566,13 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	);
 
 	// Initialize the editor.
-	wp_add_inline_script( 'wp-editor', 'wp.api.init().done( function() { wp.editor.createEditorInstance( \'editor\', window._wpGutenbergPost ); } );' );
+	$editor_settings = array(
+		'wideImages' => get_theme_support( 'wide-images' ),
+	);
+	wp_add_inline_script( 'wp-editor', 'wp.api.init().done( function() {'
+		. 'wp.editor.createEditorInstance( \'editor\', window._wpGutenbergPost, ' . json_encode( $editor_settings ) . ' ); '
```

Comment:
Above we're already creating a global object via `wp_localize_script`, used to initialize TinyMCE settings. I like in this case that we're more explicit about passing an options object to the `createEditorInstance` (vs. assuming globals from code itself), but wondering if we should merge `$editor_settings` into `wpEditorL10n` and pass this object from the corresponding property e.g. `wpEditorL10n.settings`